cli/ssh_manager.go

```diff
@@ -0,0 +1,153 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+
+	"github.com/mongodb/jasper"
+	"github.com/pkg/errors"
+)
+
+// sshManager uses SSH to access a remote machine's Jasper CLI, which has access to
+// methods in the Manager interface.
+type sshManager struct {
```

Comment:
re: conversation in person, this should wrap a jasper manager, so that we can track the ssh processes that we make remotely.

Reply:
There's still going to be an issue where the wrapped manager will just be creating `Command`s, which aren't mocked since the Command is not an interface. There's also no way to introspect on the Command implementation without a mock of it (hence the ticket I mentioned).